plone/app/locales/locales/de/LC_MESSAGES/plone.po

```diff
@@ -10643,12 +10643,12 @@ msgstr "Aktivieren Sie diese Option, wenn Sie einen Caching-Proxy vor Plone konf
 #. Default: "Enter your email and we’ll send you your username."
 #: CMFPlone/interfaces/login.py:76
 msgid "help_pwreset_email"
-msgstr "Geben Sie ihre E-Mail an und wir senden Ihnen ihren Benutzernamen."
+msgstr "Geben Sie ihre E-Mail an und wir senden Ihnen Ihren Benutzernamen."
```

Comment:
Das erste "Ihre" gross (Anrede), das Ihnen auch (Anrede) das dritte nicht (keine Anrede).
https://www.duden.de/sprachwissen/sprachratgeber/Gross-oder-Kleinschreibung-von-duDu-und-ihrIhr

Reply:
Vor lauter Anreden könnte man das auch so schreiben:
```suggestion
msgstr "Geben Sie ihre E-Mail an und wir senden Ihnen den Benutzernamen."
```
Die Frage ist ob "Benutzername" noch gegendert gehört.

Reply:
Laut Duden werden die Höflichkeitsanrede _Sie_ und das entsprechende Possessivpronomen _Ihr_ immer großgeschrieben. 
https://www.duden.de/sprachwissen/rechtschreibregeln/Gro%C3%9F-%20und%20Kleinschreibung#D84
D.h. In "Geben Sie Ihre E-Mail an und wir senden Ihnen Ihren Benutzernamen." würden alle vier Anredepronomen (Personalpronomen und Possessivpronomen) großgeschrieben.

Dennoch wäre eine geschmeidigere Formulierung prima. 

```suggestion
msgstr "Fordern Sie Ihren Benutzernamen an. Dafür tragen Sie hier bitte Ihre E-Mail-Adresse ein."
```